__tests__/unit.test.ts

```diff
@@ -26,8 +40,7 @@ describe('unit test suite', () => {
     const result = createMetricDatum(metricName, projectName, isCronJob, value)
     expect(result.MetricName).toStrictEqual(metricName)
     expect(result.Value).toStrictEqual(value)
-    expect(result.Dimensions.length).toStrictEqual(2)
```

Comment:
Why not check against the current expected number of dimensions?

Reply:
I realized this may be an example of a [change-detector test](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) which isn't useful if we want to track more metrics down the line.

Reply:
You could check that length >=1